src/python/pants/backend/jvm/targets/jvm_target.py

```diff
@@ -40,6 +41,7 @@ def __init__(self,
                services=None,
                platform=None,
                strict_deps=None,
+               exports=None,
```

Comment:
I wonder if this could even go on `Target` itself?  There's nothing intrinsically JVM-y about the concept, that I can see. The same is true for `strict_deps` itself, of course, but I wouldn't expect you to move that in this change.

Or is there some reason why it makes more sense for this to be only on JVM targets?